tests/scripts/restore-mon-quorum-using-osds.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash -ex
+
+RETRY_PERIOD=10
+RETRY_COUNT=30
+
+function usage() {
+    retval=$1
+    echo "usage: $0 [-hr] -f <fsid> -n <namespace> -b <backupdir> -t <tmpdir>"
+    echo "	-h: show this help"
+    echo "	-r: restore deployments from backup, should also set -b"
+    echo "	-f <fsid>: fsid of your cluster"
+    echo "	-n <namespace>: namespace of your cluster"
+    echo "	-b <backupdir>: a directory to store backup. it must be empty"
+    echo "	-t <tmpdir>: a directory to store temporary data during restoring"
+    exit $retval
+}
+
+function wait_pods() {
+    local appname=$1
+    local desired_count=$2
+
+    local status_option
+    if [ -n "$3" ]; then
+        status_option="--field-selector=status.phase=$3"
+    else
+        status_option=""
+    fi
+
+    sleep 5
```

Comment:
Why?

Reply:
This sleep is to prevent a possible false positive if `actual_pods == desired_pods`. For example,  depeting three mon pods, calling `wait_pods 3`, and expect all three osds are rebooted, but this condition accidentally becomes true when some pods are terminating.

Probably I should check reatarting count for this purpose. This PR is still under in progress.